android/src/main/java/com/revenuecat/purchases/hybridcommon/mappers/StoreProductMapper.kt

```diff
@@ -14,7 +14,8 @@ fun StoreProduct.map(): Map<String, Any?> =
         "introPrice" to mapIntroPrice(),
         "discounts" to null,
         "productCategory" to mapProductCategory(),
-        "productType" to mapProductType()
+        "productType" to mapProductType(),
+        "subscriptionPeriod" to subscriptionPeriod
```

Comment:
I think we don't have many tests on these mappers but I wonder if we should add them... Not a blocker for this PR though I guess.